### PR TITLE
Move database copying into its own function in main migration script

### PR DESCRIPTION
**Pull request by plaes**
Small change to make the main migration script a bit more organized.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Edit: One minor PEP8 thing

File: scripts/migrate.py [hunk 1]
```diff
@@ -14,6 +14,50 @@
     print ('couldn\'t import bzrlib. You won\'t be able to run migrations '
            '<= 7.0!')
 
+def copy_database(conn_parms):
```
Comment (pedrobaeza) [inline scripts/migrate.py:18]: There must be two blank lines, not one.

**Comment by legalsylvain** [comment]
:+1: thanks for the refactoring. 
code review, no test.

**Comment by plaes** [comment]
Should I squash and push-force this branch again?

**Comment by pedrobaeza** [comment]
No, it's not needed, only with the new commit Travis makes again the tests, and as you can see, now is green.

Thanks :+1:

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**